packages/design-system/src/elements/toniq-input/toniq-input.element.ts

```diff
@@ -196,6 +199,10 @@ export const ToniqInput = defineToniqElement<{
             input::placeholder {
                 color: ${toniqColors.accentTertiary.foregroundColor};
             }
+
+            .suffix {
+                font-weight: 500;
```

Comment:
Despite what the design say, let's go with using the bold style here so we're utilizing `toniqFontStyles`:

```suggestion
                ${toniqFontStyles.boldFont};
```